db/get_kitchens_by_id.sql

```diff
@@ -0,0 +1,5 @@
+SELECT * 
+FROM kitchen 
+JOIN kitchen_users ON kitchen.id = kitchen_users.kitchen_id 
+JOIN users ON users.id = kitchen_users.user_id 
+WHERE user_id = ${id};
```

Comment:
Quick note on this query - the name is "get_kitchens_..." **but** the query is actually doing `get_kitchens_and_users`. This much information might be unnecessary. Do you need to return the results from kitchen, kitchen_users, and users, or just kitchen?